modules/openapi-generator/src/main/resources/Javascript-Apollo/ApiClient.mustache

```diff
@@ -53,7 +53,7 @@ export default class ApiClient extends RESTDataSource {
     }
 
     parametrizePath(path, pathParams) {
-        return url.replace(/\{([\w-]+)\}/g, (fullMatch, key) => {
```

Comment:
I have no idea what `url` referred to — there's no such variable! Switching to `path` appears to work (though it's a bit of a blind solve).